app/views/dashboard/_stripe_account_setup_banner.njk

```diff
@@ -35,5 +58,15 @@
       }}
       </div>
     </div>
+  {% elif  stripeAccountDetails.payouts_enabled === false %}
```

Comment:
Now that we've got a few entities all "something" accounts I think it might be useful to be particularly clear with our naming (both in the controller and the template). 

Some ideas: 
`account` -> `gatewayAccount`
`stripeAccountDetails` -> `stripeAccount`
`stripeAccountSetup` -> `connectorGatewayAccountStripeProgress`

Reply:
Done